ext/nnstreamer/tensor_converter/tensor_converter_flatbuf.cc

```diff
@@ -108,12 +110,14 @@ fbc_convert (GstBuffer *in_buf, GstTensorsConfig *config, void *priv_data)
     std::string _name = tensor->Get (i)->name ()->str ();
     const gchar *name = _name.c_str ();
 
-    config->info.info[i].name = (name && strlen (name) > 0) ? g_strdup (name) : NULL;
-    config->info.info[i].type = (tensor_type) tensor->Get (i)->type ();
+    _info = gst_tensors_info_get_nth_info (&config->info, i);
+
+    _info->name = (name && strlen (name) > 0) ? g_strdup (name) : NULL;
```

Comment:
If ```name``` is NULL, ```g_strdup()``` returns NULL..

Reply:
Now we initialize the tensor name in tensors-info struct as null, and do not set this if a tensor does not use its name. I am not sure but this may cause unexpected error.
If necessary we need considering another PR to change default value of tensor name.